examples/run_smc_on_freesolv.py

```diff
@@ -1,17 +1,28 @@
 import argparse
```

Comment:
Based on offline discussion, makes sense to move the other smc example to the attic (either in this PR or I can do so in a later PR)